nautobot/docs/development/core/getting-started.md

```diff
@@ -514,7 +514,7 @@ Quit the server with CONTROL-C.
 
 Please see the [official Django documentation on `runserver`](https://docs.djangoproject.com/en/stable/ref/django-admin/#runserver) for more information.
 
-You can connect to the development server at `localhost:8080`.
+You can connect to the development server by running `invoke open-nautobot-web`.
```

Comment:
Personally I'd prefer that this still recommend localhost:8080, with a note about the existence of the invoke command and the requirement to use it if you've enabled ephemeral ports.